enhancements/audit-command.md

```diff
@@ -0,0 +1,843 @@
+---
+title: audit-command-operation
+authors:
+  - "@camilamacedo86"
+reviewers:
+  - TBD
+approvers:
+  - TBD
+creation-date: 2021-02-22
+last-updated: 2021-03-02
+status: implementable
+see-also:
+  - [custom operator validation #43](https://github.com/operator-framework/enhancements/pull/43)
+  - [Enhancement for validating bundles in a catsrc image #48](https://github.com/operator-framework/enhancements/pull/48)
+  - [operator-framework/api: add new checks to the operatorhub validator][ep-new-checks for-operator-hub]
+  - [New OpenShift Validator][ocp-validator-proposal]
+---
+
+# Audit command operation
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [x] Design details are appropriately documented from clear requirements
+- [x] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+
+## Open Questions 
+
+1. Should this command be added into SDK or OPM or in a new tool?
+
+Beside the [catalog builds in operator-sdk](operator-sdk-builds-catalogs.md) proposal, SDK maintainers (@jesus, @estroz) point out that they think that it would better fit in OPM and not in SDK and that while the SDK has some catalog abilities in that EP see that t has been using OPM binary as dependency to achieve it and the [non-goal](https://github.com/operator-framework/enhancements/blob/master/enhancements/operator-sdk-builds-catalogs.md#non-goals).
+ 
+However, this proposal aims to discuss the audit command and how and what it will do and not exactly where it will live. This decision is out of this proposal's scope since it brings strategy decisions that raise questions like: Do we want more than one tool for the framework? We will have a tool kit that helps out users with the framework solutions or many for each purpose? What are the advantages and facilities or not for the users and maintainers in all scenarios? What is the vision for OPM and SDK tools as Operator Framework in the future?. In this way, let's keep it open until we are able to properly defined all that should be done by audit and how as to have the resources required to address and implement this proposal.
```

Comment:
```suggestion
However, this proposal aims to discuss the audit command and how and what it will do and not exactly where it will live. This decision is out of this proposal's scope since it brings strategy decisions that raise questions. In this way, let's keep it open until we are able to properly defined all that should be done by audit and how to have the resources required to address and implement this proposal.
```